diff -Nur -x '*.orig' -x '*~' trunk/nautilussvn/lib/extensions/nautilus/NautilusSvn.py trunk.new/nautilussvn/lib/extensions/nautilus/NautilusSvn.py
--- trunk/nautilussvn/lib/extensions/nautilus/NautilusSvn.py	2009-05-24 16:29:05.000000000 +0800
+++ trunk.new/nautilussvn/lib/extensions/nautilus/NautilusSvn.py	2009-05-28 23:14:39.000000000 +0800
@@ -524,7 +524,7 @@
     
     """
     
-    SEPARATOR = "- - - - - - - - - - - - - - -"
+    SEPARATOR = u'\u2015' * 10
     
     def __init__(self, paths, nautilussvn_extension):
         self.paths = paths
