diff -Nur -x '*.orig' -x '*~' rabbitvcs-0.12/setup.py rabbitvcs-0.12.new/setup.py
--- rabbitvcs-0.12/setup.py	2009-09-28 20:11:45.000000000 +0800
+++ rabbitvcs-0.12.new/setup.py	2009-09-28 20:18:56.000000000 +0800
@@ -125,10 +125,11 @@
 )]
 
 # Update notifier
-update_notifier = [("/usr/share/rabbitvcs", [
-    "packages/ubuntu/debian/rabbitvcs-restart-required.update-notifier",
-    "packages/ubuntu/debian/do-rabbitvcs-restart-nautilus"
-])]
+# Not used for Debian package
+# update_notifier = [("/usr/share/rabbitvcs", [
+#     "packages/ubuntu/debian/rabbitvcs-restart-required.update-notifier",
+#     "packages/ubuntu/debian/do-rabbitvcs-restart-nautilus"
+# ])]
 
 # Documentation
 documentation = [("/usr/share/doc/rabbitvcs", [
@@ -169,7 +170,7 @@
             "ui/glade/*.glade"
         ]
     },
-    data_files=nautilus_extension + command_line_tool + translations + icons + documentation + update_notifier + config_spec
+    data_files=nautilus_extension + command_line_tool + translations + icons + documentation + config_spec # + update_notifier
 )
 
 #
@@ -177,10 +178,11 @@
 #
 
 # Make sure the icon cache is deleted and recreated
-if sys.argv[1] == "install":
-    print "Running gtk-update-icon-cache"
-    
-    subprocess.Popen(
-        ["gtk-update-icon-cache", icon_theme_directory], 
-        stdout=subprocess.PIPE
-    ).communicate()[0]
+# Debian: this is not needed since hicolor-icon-cache (0.10-2)
+# if sys.argv[1] == "install":
+#     print "Running gtk-update-icon-cache"
+#     
+#     subprocess.Popen(
+#         ["gtk-update-icon-cache", icon_theme_directory], 
+#         stdout=subprocess.PIPE
+#     ).communicate()[0]
