diff -Nur -x '*.orig' -x '*~' RVCS-pkg/setup.py RVCS-pkg.new/setup.py
--- RVCS-pkg/setup.py	2009-09-28 20:11:45.000000000 +0800
+++ RVCS-pkg.new/setup.py	2009-10-05 22:12:08.000000000 +0800
@@ -175,12 +175,12 @@
 #
 # Post installation
 #
-
+# Ubuntu Karmic: this is not needed since hicolor-icon-cache (0.10-2)
 # Make sure the icon cache is deleted and recreated
-if sys.argv[1] == "install":
-    print "Running gtk-update-icon-cache"
-    
-    subprocess.Popen(
-        ["gtk-update-icon-cache", icon_theme_directory], 
-        stdout=subprocess.PIPE
-    ).communicate()[0]
+# if sys.argv[1] == "install":
+#     print "Running gtk-update-icon-cache"
+#     
+#     subprocess.Popen(
+#         ["gtk-update-icon-cache", icon_theme_directory], 
+#         stdout=subprocess.PIPE
+#     ).communicate()[0]
